### ISSUE TITLE
Add a option to disable to app pin code.

### ISSUE THREAD

**Issue by MrMuffinIsTaken** [raivo-otp/ios-application#105]
**Is your feature request related to a problem? Please describe.**
Yes and no, i'm kind of annoyed at the extra pin code requirement on the TOTP app i use seen as i don't want my TOTP authenticator app to force me to need to use an additional pin code on it. This is something that should rather be an opt in, then an op out by default and worst of all is that i can't even opt out of it (to my knowledge).

**Describe the solution you'd like**
I would like for it to be an option in the settings that one can enable it or disable, not a requirement that will turn me or anyone else away for the app. 

**Describe alternatives you've considered**
I haven't considered switching to anther app yet, but if this still is a requirement in the near future. Then i may consider alternatives.

**Additional context**
I would like the pin code option to be an opt in feature and i would like to be mentioned on setup, to where if you would like to enable it or disable it.

**Comment by Shravansingh** [comment]
I want the same thing. I forgot my pin code and now I am stuck. I do have my master password with me. Is there a way to access my account with my master password. Please help?

**Comment by MrMuffinIsTaken** [comment]
> I want the same thing. I forgot my pin code and now I am stuck. I do have my master password with me. Is there a way to access my account with my master password. Please help?

I don't think so, you probably have to make a backup, delete the app, and restore from that one backup. @tijme will know, but that's what i think needs to be done.

**Comment by sinanmb** [comment]
It would be great to be able to disable the pin code and rely on Face ID instead.
